.github/workflows/build-and-push.yml

```diff
@@ -8,17 +8,17 @@ jobs:
   build-and-push:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        version: [1.18, 1.20]
         include:
-          - dockerfile: "Dockerfile"
+          - version: "1.18"
+            dockerfile: "Dockerfile"
+            registry_namespace: "centos7"
+            tag: "centos7"
+          - version: "1.20"
+            dockerfile: "Dockerfile"
             registry_namespace: "centos7"
-            registry_suffix: "-centos7"
             tag: "centos7"
-          - dockerfile: "Dockerfile.c9s"
```

Comment:
Why the c9s option is removed in this commit? Does it have an impact on building and pushing Nginx 1.20 container?